refinery/analysis_manager/schemas/AnalysisConfig.json

```diff
@@ -0,0 +1,92 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "oneOf": [
+    {
+      "required": [
+        "custom_name",
+        "studyUuid",
+        "toolUuid",
+        "user_id",
+        "workflowUuid"
+      ],
+      "properties": {
+        "custom_name": {
+          "type": "string"
+        },
+        "studyUuid": {
+          "type": "string",
+          "pattern": "^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$"
```

Comment:
Can you use the [`$ref`](https://spacetelescope.github.io/understanding-json-schema/structuring.html#reuse) syntax rather than copy and pasting this?